PEP 458: Rename Role and Key Terms According to Issue #13

vladimir-v-diaz opened the pull request:
Addresses Issue #13.

* Update the figures and text to use the new terms.  `Online / offline` instances have been renamed to `on-pypi / off-pypi`, and `claimed / unclaimed` to `pypi-signed / developer-signed`.

* Expanded the section that discusses the `lazy bin walk` scheme.

lvigdor commented on pep-0458-tuf-online-keys.rst:
```diff
@@ -114,10 +114,10 @@ Threat Model
 
-* Attackers can compromise at least one of PyPI's trusted keys stored online,
-  and may do so at once or over a period of time.
+* Attackers can compromise at least one of PyPI's trusted keys stored on PyPI
```
there should be a "the" after "stored on"

lvigdor commented on pep-0458-tuf-online-keys.rst:
```diff
@@ -505,34 +506,41 @@ Metadata Scalability
+if the *pypi-signed* role itself signed for about 220K PyPI targets (which are
+simple indices and distributions).  This PEP does not delve into the details,
+but TUF features a so-called "`lazy bin walk`__" scheme that splits a large
+targets' metadata file into many small ones (bins).  Targets are then
```
514: "targets' " should either be "targets" or "target's"

lvigdor commented on pep-0458-tuf-online-keys.rst:
568: seems odd "compromise one key from automatically compromising the rest of the keys"
I think the first compromise should be replaced with "prevent"

lvigdor commented on pep-0458-tuf-online-keys.rst:
```diff
@@ -580,27 +589,27 @@ administrators or all PSF board members, and t > 1 (so that at least two
-In order to support continuous delivery, the *timestamp*, *snapshot*, *bins*
-role keys MUST be online.
+In order to support continuous delivery, the *timestamp*, *snapshot*,
+*pypi-signed* role keys MUST be stored on PyPI infrastructure (on-pypi keys).
```
add "the" before PyPI

vladimir-v-diaz commented:
@lvigdor 
Updates addressing your comments were made here: https://github.com/vladimir-v-diaz/interoperability-peps/commit/474c2f75d9c64b95d12f1c3f72de9d4bc7c1b509
Thanks for reviewing this PR!
